src/components/Contentful/Highlight.js

```diff
@@ -34,7 +35,7 @@ export default function ContentfulHighlight({
     colClasses += ` col-${extraSmallColumnWidth}`
   }
 
-  const containerClassName = `contentful-highlight`
+  const containerClassName = `contentful-highlight ` + customClasses
```

Comment:
```suggestion
  const containerClassName = `contentful-highlight ${customClasses}`
```
More of a stylist opinion rather than any required changes. Up to you on whether to change or not 👍